spec/decorators/visit_decorator_spec.rb

```diff
@@ -35,4 +35,76 @@
       end
     end
   end
+
+  describe '#book_to_nomis_opt_in?' do
+    let(:nomis_checker) { instance_double(StaffNomisChecker) }
+
+    before do
+    end
+
+    describe 'when the book to nomis feature is disabled' do
+      before do
+        switch_off :nomis_staff_book_to_nomis_enabled
+      end
+
+      it { expect(subject.book_to_nomis_opt_in?).to eq(false) }
```

Comment:
The use of predicate methods has the downside that you lose the context of the exact method call. I think the tests end up being more maintainable by using less RSpec sugar methods, my preference would even to use mini_test!